src/noui.h

```diff
@@ -5,6 +5,8 @@
 #ifndef BITCOIN_NOUI_H
 #define BITCOIN_NOUI_H
 
+#include <boost/signals2/connection.hpp>
```

Comment:
I don't think these includes need to be added here, better include them in the `cpp` file if they're used there to avoid the boost stuff having to be processed for every compilation unit.

Reply:
Adjusted